saw-core-coq/src/Verifier/SAW/Translation/Coq/Term.hs

```diff
@@ -666,26 +673,29 @@ translateTermToDocWith ::
   TranslationReader ->
   [String] ->
   [String] ->
-  (Coq.Term -> Doc ann) ->
-  Term ->
+  (Coq.Term -> Coq.Term -> Doc ann) ->
```

Comment:
Worth stating in a comment what these two `Term`s represent, since it's not obvious unless you stare at the code a bit.

Reply:
Ooh, good point. In fact, there probably needs to be even more haddock as well...

Reply:
Does Haddock now support `-- ^` after the arrow correctly?